### PR TITLE
Conformance results for v1.18/QFusion

### PR DESCRIPTION
**Pull request by matt88s**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [x] Did you include the product/project logo in SVG, EPS or AI format?  
* [x] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] If your product/project is open source, did you include the `repo_url`?
* [x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

**Comment by matt88s** [comment]
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [x] Did you include the product/project logo in SVG, EPS or AI format?  
* [x] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [ ] If your product/project is open source, did you include the `repo_url`?
* [ ] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

File: v1.18/qfusion/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## QFusion
+QFusion is an enterprise-level DBaaS platform based on cloud native technologies which support for MySQL,SQL Server,Redis,Oracle,PostgreSQL and MongoDB database.It implements key components of database operator orchestration and provides self-healing and auto-scale database feature
+
+## Reproduce Conformance Tests
+###  prepare
+`contact [Woqutech](http://woqutech.com)  to download relvent images and push to own registry`
```
Comment (ErinThacker) [inline v1.18/qfusion/README.md:6]: Please include installation instructions, please include detailed instructions for installing/configuring your product - they cannot just include a link to their instructions (to minimize link rot etc)

Comment (ErinThacker) [reply v1.18/qfusion/README.md:6]: Following up on request for installation instructions.

Comment (matt88s) [reply v1.18/qfusion/README.md]: installation instruction update in README.md

**Comment by ErinThacker** [comment]
You are now Certified Kubernetes.

**Pull request merged**